src/pages/_sections/FeatureProjects.tsx

```diff
@@ -10,28 +9,41 @@ interface FeatureProjectsProps {
   className?: string;
 }
 export const FeatureProjects = ({ className }: FeatureProjectsProps) => {
+  const { projects, isLoading } = useProjects();
+
+  const projectsSlice = projects.sort(() => 0.5 - Math.random()).slice(0, 4);
```

Comment:
🟠 `.sort` is mutating the original `projects` array, which could produce undesired results across other usages - I suggest using `.toSorted` (if available in the current TS version), or spreading `projects` within an in-lined array as so:
```suggestion
  const projectsSlice = [...projects].sort(...)
```

Reply:
Damn... Thanks for pointing that out.